your-first-cluster.md

```diff
@@ -0,0 +1,512 @@
+This guide covers setting up and using Cluster API using Docker infrastructure, - which is the same way CAPI runs its end-to-end tests. To set up a Cluster using AWS, Azure GCP and many more see the [CAPI quick-start guide](https://cluster-api.sigs.k8s.io/user/quick-start.html)
+
+- [Linux](#Linux)
+    - [The management cluster](#the-management-cluster)
+    - [Your first workload cluster](#your-first-workload-cluster)
+- [macOS](#MacOS)
+    - [The management cluster](#the-management-cluster-1)
+    - [Your first workload cluster](#your-first-workload-cluster-1)
+- [Windows](#Windows)
+    - [The management cluster](#the-management-cluster-2)
+    - [Your first workload cluster](#your-first-workload-cluster-2)
+
+
+# Linux 
+
+## The Management Cluster
+
+A Cluster API Management Cluster is a Kubernetes Cluster where the components that make up the CAPI control plane are installed. These components include the Custom Resources used to create and manage a cluster and the controllers that operate on them.
+
+The first step in creating a management cluster is to create a Kubernetes cluster to host the CAPI components.
+
+```bash
```

Comment:
Q: I would recommend to have the prepull script here as well

Mostly because the flow won't work if the images are not pre-pulled